paper/paper.md

```diff
@@ -0,0 +1,88 @@
+---
+
+<!-- pandoc -s paper.md -o paper.pdf --bibliography paper.bib --csl=apa.csl --filter pandoc-citeproc --pdf-engine=xelatex-->
+
+# Summary
+
+**ASSUME** is an open-source toolbox for agent-based simulations of various energy markets, with a primary focus on European electricity markets. Developed as an open-source model, its objectives are to ensure usability and customizability for a wide range of users and use cases in the energy system modeling community.
+
+A unique feature of the `ASSUME` toolbox is its integration of **Deep Reinforcement Learning** methods into the behavioral strategies of market agents. The model offers various predefined agent representations for both the demand and generation sides, which can be used as plug-and-play modules, simplifying the usage of reinforcement learning strategies. This setup enables research of new market designs and evolving dynamics in energy markets.
+
+# Statement of need
+
+While other open-source agent-based models have been developed for studying energy markets, such as PowerACE [@bublitzAgentbasedSimulationGerman2014] and AMIRIS [@schimeczekAMIRISAgentbasedMarket2023], the possible integration of reinforcement learning methods into the behavioral strategies of market agents is currently unique to `ASSUME`.
+Most notably it enables the highest number of simultanelously learning market agents in literature [@miskiwExplainableDeepReinforcement2024].  
+This feature allows for the exploration of new market designs and emergent dynamics in energy markets using a common open-source simulation framework.
+Further unique features of `ASSUME` are the extensive market abstraction which allows to define complex multi-market scenarios as shown in [@maurerMarketAbstractionEnergy2023].
+`ASSUME` is designed to provide results which are easily comparable and interactable.
+This is set in contrast to other agent-based simulation frameworks, which are often designed to be used for a specific use case.
+Instead, various moving parts can be configured, such as the market abstraction, the agent properties and bidding strategies, or the reinforcement learning methods.
+
+# Architecture
+
+`ASSUME` builds on the open-source agent-based simulation library [mango-agents](https://pypi.org/project/mango-agents/) to model the simulation world and to give users continuous feedback during the simulation runs.
+Interaction with the results of the simulation is possible through the Grafana dashboards, which do have direct access to the simulation database.
+Additionally, writing the output to CSV files is supported for scenarios in which a database is not available or needed.
+
+New scenarios can be created by providing CSV files, accessing simulation data from a database or by using the object-oriented API, which makes it possible to integrate new classes, maket mechanisms and bidding strategies as well.
+
+![Basic Architecture of the ASSUME simulation\label{fig:architecture}](../docs/source/img/architecture.svg)
```

Comment:
I would not call it basic, but rather Simplified or Componental overview of the ASSUME architecture